index.html

```diff
@@ -33,6 +33,293 @@
 </head>
 
 <body>
+  <section id="abstract">
+    <h2>Abstract</h2>
+    <p>
+      This document defines a new API {{MediaDevices/getAllScreenMedia}}
+      for capturing multiple
+      <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>
+      following one user gesture.
+      It is an extension to the Screen Capture API [[screen-capture]].
+    </p>
+  </section>
+  <section id="sotd">
+    <p>
+      This document is a draft. It is subject to major changes and, while early
+      experimentations are encouraged, it is therefore not intended for implementation.
+    </p>
+  </section>
+  <section class="informative" id="introduction">
+    <h2>
+      Introduction
+    </h2>
+    <p>
+      The Screen Capture API [[screen-capture]] enables the capturing of a single
+      <a data-cite="screen-capture#dfn-display-surface">display surface</a> in the
+      form of a video track.
+    </p>
+    <p>
+
+      Users currently cannot capture multiple
+      <a data-cite="screen-capture#dfn-monitor">monitors</a>
+      at once without a [=transient activation=]. This may be useful in certain
+      scenarios, e.g. for compliance or training purposes.
```

Comment:
It's uncelar why a single transient activation would be a show-stopper for compliance and training purposes. Surely the end-user can be relied upon to click one button? But if you explain the serpentine flow of interactions with the page, browser, page, browser...

Reply:
revised the whole paragraph.